content/pipes.md

```diff
@@ -315,6 +331,8 @@ async create(createCatDto) {
 
 > info **Hint** The `@UsePipes()` decorator is imported from the `@nestjs/common` package.
 
+> info **Note** When a DTO already uses `class-validator` to validate each data element, the techniques above using both Joi schema and a DTO are not necessary. You only need one level of validation. However, if your DTO does not have `class-validator` in use, then the Joi schema method could be used like the above example, without worrying about breaking "DRY" principles.
```

Comment:
now that we know how that DTO looks like, we don't this this note anymore. Another issue with it is that it mention the integration with `class-validator`, which wasn't describe at this point

```suggestion
```

Reply:
:ping_pong:  @carbondigitalus

Reply:
Sorry, I thought you guys were going to update it.

Reply:
@jared-leddy [Step-by-step guide to contributing on GitHub](https://www.dataschool.io/how-to-contribute-on-github/)